src/components/Header/Header.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { ReactComponent as Arrow } from '../Design/img/arrow-left.svg'
+import { ReactComponent as Heart } from '../Design/img/heart.svg'
+import { ReactComponent as Profile } from '../Design/img/user-header.svg'
+import styled from 'styled-components/macro'
+import { useHistory, Link } from 'react-router-dom'
+
+export default function Header() {
+  const history = useHistory()
+  return (
+    <div>
+      <ArrowStyled onClick={handleClick} />
+      <Link to="/favorites">
```

Comment:
You could also use NavLink here. Would be useful, if you want to highlight which page is active.